docs/options.md

```diff
@@ -12,3 +12,4 @@ There are a couple of driver options that can be passed as arguments when starti
 | logging-format              | json                                              | text                                                | Sets the log format. Permitted formats: text, json|
 | user-agent-extra            | csi-ebs                                           | helm                                                | Extra string appended to user agent|
 | enable-otel-tracing         | true                                              | false                                               | If set to true, the driver will enable opentelemetry tracing. Might need [additional env variables](https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/#general-sdk-configuration) to export the traces to the right collector|
+| batching                    | true                                              | true                                                | If set to true, the driver will enable batching of API calls. This is especially helpful for improving performance in workloads that are sensitive to EC2 rate limits|
```

Comment:
Additional comment: Can we add something like "at the cost of a small increase to worst-case latency"? Just so customers that want to hyper-optimize latency understand the impact of the option.

Reply:
✅